<launch>
    <node name="search_action_server" pkg="team34" type="search_server.py" output="screen">
    </node>
    <node name="search_action_client" pkg="team34" type="search_client.py" output="screen">
    </node>
</launch>